src/toArray.js

```diff
@@ -0,0 +1,12 @@
+const toArray = input => {
+  let result = []
+  if (input instanceof String) {
+    return input.split('')
+  } else if (input instanceof Object) {
+    return input.values()
```

Comment:
Does this work for inputs that are `Array`s already? We should probably create a test to verify the behavior in this case.